### DISCUSSION TITLE
Drag and drop with network

### DISCUSSION THREAD

**Discussion by laurencefloriani** [airbnb/visx#1455] [Q&A]
Is it possible add drag and drop function in a network component ?

**Comment by williaster** [comment]
should be possible! I'd check out [this example](https://airbnb.io/visx/drag-i), note there's also a `useDrag` hook in the `@visx/drag` package https://airbnb.io/visx/docs/drag#useDrag

`@visx/network` really doesn't handle layout at all, which I think would be the hard part with any network implementation.
